### PR TITLE
Add MDNEXT to projects page

### PR DESCRIPTION
**Pull request by domitriusclark**
MDNEXT is an ecosystem of tools and templates surrounding the MDX + NextJS ecosystem. It includes a growing component library, a handful of templates, and a CLI for choosing one!

### CODE REVIEW

**Comment by wooorm** [comment]
Hmm, this doesn’t seem to do what you intend to do?

The thing `[x]: y`, is a link definition, but the link isn’t used anywhere?

**Comment by wooorm** [comment]
@domitriusclark Ping!

File: docs/projects.mdx [hunk 1]
```diff
@@ -15,6 +15,7 @@
   Spectacle
 - [Charge][]: An opinionated, zero-config static site generator
```
Comment (johno) [inline docs/projects.mdx:17]: Below charge you can add the following to use the link reference you've added below:

```mdx
- [MDNEXT][]: INSERT DESCRIPTION HERE
```

**Comment by wooorm** [comment]
@domitriusclark Anything I can do to help?

**Pull request merged**